MaterialLib/MPL/MaterialSpatialDistributionMap.cpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * \file
```

Comment:
I see, but this is a little trickier: `\file` should be there, otherwise doxygen puts documentation from different files together like [here](https://doxygen.opengeosys.org/d8/df9/GroundwaterFlowFEM_8h.html), see namespace section. The `\file` is missing in https://github.com/ufz/ogs/blob/master/ProcessLib/TwoPhaseFlowWithPP/TwoPhaseFlowWithPPLocalAssembler-impl.h

Filename is optional and is the current file name, whatever it is. Adding explicitly a file name requires also an update of the `\file xxx.h` to `\file yyy.h` when moving files.

Reply:
You are right about the blank line, thanks! Will change.

Reply:
I tried four options:
1. no `\file`
2. `\file` followed by `\author` on new line.
3. `\file` followed by empty line.
4. `\file` followed by XXX on new line.
5. `\file` followed by XXX on same line.

Number 1 and 5 do not produce correct output. So, it seems, that new line is sufficient. Will keep the original version (2).

Reply:
or \file followed by the file name.

Reply:
Adding explicitly a file name requires also an update of the \file xxx.h to \file yyy.h when moving files.

Reply:
I see.